d2l-typography-shared-styles.html

```diff
@@ -26,6 +26,14 @@
 					text-transform: uppercase;
 				};
 
+				--d2l-body-compact-text: {
+					color: var(--d2l-color-ferrite);
+					font-family: inherit;
+					font-size: 0.8rem;
+					font-weight: 400;
+					line-height: 1rem;
```

Comment:
You're sure about the line-height value here? Remember all the problems we've had with setting it to anything other than `normal`...